easy-ui-react/src/IconButton/IconButton.module.scss

```diff
@@ -0,0 +1,26 @@
+@import "../styles/common";
+
+.IconButton {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: var(--ezui-shape-border-radius-base);
+  cursor: pointer;
+
+  &:focus {
+    box-shadow: var(--ezui-shadow-button);
+    outline: none;
+  }
+
+  &:disabled {
+    pointer-events: none;
+  }
+}
```

Comment:
**thought:** i notice that most of these styles are used across the buttons. would it make sense to create a mixin for the shared root styles in `Button.module.scss` to `@use` in `DropdownButton.module.scss` and `IconButton.module.scss`, similar to how we would create mixins for the variants and colors?

Reply:
agree!